graph-migration/README.md

```diff
@@ -40,20 +54,20 @@ and then submit to Spark:
 dse spark-submit target/scala-2.11/graph-migration_2.11-0.1.jar <legacy_graph> <native_graph>
 ```
 
-
 Example:
 ```
 dse spark-submit target/scala-2.11/graph-migration_2.11-0.1.jar reviewerRating nreviewerRating
 ```
 ### Development notes
 
-The migration tool use spark-cassandra-connector to directly write data to DSE-DB. It enumerates native vertex labels, 
-select proper rows from legacy DGF vertices and write them to  appropriate tables. See `migrateVertices()` method
-Then the tool enumerate native edge labels, extract (inLabel, edgeLabel, outLabel) triplet from it, select proper rows
-from legacy DGF edges, convert edge ids into native format and write data to corresponded table. See `migrateEdges()` method.
-`handleMultiAndMetaProperies()` method is expected to be overridden to properly migrate multi and meta properties
-    by default it just drops all  metadata and select the first of multi properties.                                  
-`getLegacyPropertyName()` should be overridden if property names was manually changed in the schema generate by the 
-   `graph-migrate-schema`
-   
- 
\ No newline at end of file
+The migration tool uses spark-cassandra-connector to directly write data to DSE-DB.
```

Comment:
#2 introduce new update API usage instead of direct used of spark-cassandra-connector

Reply:
so you can remove this sentence

Reply:
Okay, updated accordingly.